src/coreclr/jit/emitarm64.cpp

```diff
@@ -1397,6 +1397,34 @@ static const char * const  bRegNames[] =
 
 // clang-format on
 
+//------------------------------------------------------------------------
+// emitPacInProlog: Sign LR as part of Pointer Authentication (PAC) support
+//
+void emitter::emitPacInProlog()
+{
+    if (JitConfig.JitPacEnabled() == 0)
+    {
+        return;
+    }
+    // TODO-PAC: should be paciasp
+    emitIns(INS_paciaz);
+    emitComp->unwindPacSignLR();
+}
+
+//------------------------------------------------------------------------
+// emitPacInEpilog: unsign LR as part of Pointer Authentication (PAC) support
+//
+void emitter::emitPacInEpilog()
+{
+    if (JitConfig.JitPacEnabled() == 0)
+    {
+        return;
+    }
+    // TODO-PAC: should be autiasp
```

Comment:
just to confirm, using `paciasp` and `autiasp` will happen in future PR(s), right?

Reply:
Yes. I'll remove these todos. We can search for `paciaz` to find these places.